OrbitLinuxTracing/PerfEventOpen.h

```diff
@@ -93,6 +93,12 @@ static constexpr uint64_t SAMPLE_REGS_USER_ALL =
 static constexpr uint64_t SAMPLE_REGS_USER_SP_IP =
     (1lu << PERF_REG_X86_SP) | (1lu << PERF_REG_X86_IP);
 
+// This must be in sync with struct perf_event_uret_probe in
+// PerfEventRecords.h.
+static constexpr uint64_t SAMPLE_REGS_URET_PROBE = (1lu << PERF_REG_X86_SP) |
```

Comment:
I would like this to be called `SAMPLE_REGS_USER_AX_SP_IP`, for consistency with the other ones. The names of these constants represent what they are, not what they are used for.

Reply:
It will probably be something like `SAMPLE_REGS_USER_SP_IP_ARGS`.

Reply:
cool :-)